project.go

```diff
@@ -44,6 +44,9 @@ type Projects interface {
 
 	// AddTagBindings adds or modifies the value of existing tag binding keys for a project.
 	AddTagBindings(ctx context.Context, projectID string, options ProjectAddTagBindingsOptions) ([]*TagBinding, error)
+
+	// DeleteTagBindings removes all existing tag bindings for a project.
+	DeleteTagBindings(ctx context.Context, projectID string) error
```

Comment:
Can we call this `DeleteAllTagBindings` just in case we ever add the ability to delete specific ones?